app/routes/api/emails/sendManagerReminder.jsx

```diff
@@ -2,28 +2,46 @@
 // API routing doesn't work if we export this as default.
 
 import { sendManagerReminder } from 'app/controllers/emails/sendManagerReminder';
+import { sendEmployesReminder } from 'app/controllers/emails/sendEmployeesReminder';
 import { json } from '@remix-run/node';
-import { sendManagersEmailReminder } from 'app/config/flags.json';
+import { sendManagersEmailReminder, sendEmployeesEmailReminder } from 'app/config/flags.json';
 import validateKey from 'app/utils/backend/api/validateKey';
 
 export const loader = () => {
   throw new Response(null, { status: 404, statusText: 'Not Found' });
 };
 
 export const action = async ({ request }) => {
-  const { error: keyValidationError } = validateKey(request);
+  try {
+    let managerEmails = 0;
+    let employeeEmails = 0;
+    const { error: keyValidationError } = validateKey(request);
 
-  if (keyValidationError) {
-    return json({ success: false, detail: keyValidationError }, 400);
-  }
+    if (keyValidationError) {
+      return json({ success: false, detail: keyValidationError }, 400);
+    }
+
+    if (sendManagersEmailReminder || sendEmployeesEmailReminder) {
+      if (sendManagersEmailReminder) {
+        const { error, emailsQueue } = await sendManagerReminder();
+        managerEmails = emailsQueue.length;
+        if (error) {
+          return json({ success: false, detail: ` sendManagerReminder -  ${error}` }, 500);
+        }
+      }
 
-  if (sendManagersEmailReminder) {
-    const { error, emailsQueue } = await sendManagerReminder();
-    if (error) {
-      return json({ success: false, detail: error }, 500);
+      if (sendEmployeesEmailReminder) {
+        const { error, emailsQueue } = await sendEmployesReminder();
+        employeeEmails = emailsQueue.length;
+        if (error) {
+          return json({ success: false, detail: ` sendEmployesReminder -  ${error}` }, 500);
+        }
+      }
+      return json({ success: true, employeeEmails, managerEmails }, 200);
```

Comment:
Could you we make sendEmployeesReminder a different endpoint? Queueing up emails and sending them all in one endpoint could cause timeouts, think we should try to make them apart to avoid this. We would need to create a new scheduler for terraform though, I can help with this part.

Reply:
Okay, Let's create another endpoint 👍

Reply:
@GeorginaFraire Nope, should be okay with same key. Just validating same way